KomaMRIFiles/test/runtests.jl

```diff
@@ -82,8 +82,11 @@ using TestItems, TestItemRunner
         obj1 = brain_phantom2D()
         Ns = length(obj1)
         K = 10
+        t_start = 0.0
+        t_end = 1.0
         obj1.motion = ArbitraryMotion(
-            [1.0],
+            t_start,
+            t_end,
```

Comment:
The tests for each type of motion should be inside `@testset`'s with corresponding names.

Reply:
Solved in [ed0ad5e](https://github.com/JuliaHealth/KomaMRI.jl/pull/408/commits/ed0ad5eb6e71302c5fb523b00ce744f07de5e416) and [4010c2d](https://github.com/JuliaHealth/KomaMRI.jl/pull/408/commits/4010c2de0b0e89c52d4ee1a679aa40fdf146380b)